### ISSUE TITLE
Problem on Building Serving Image with Multiple Models with Config

### ISSUE THREAD

**Issue by gabbygab1233** [tensorflow/serving#1918]
I don’t have a problem when creating my own serving image in docker using 1 model but when I try to build a serving image with multiple models it doesn’t work.

Here’s the command I used:

## **Create Serving Image**
```
docker run -d --name serving_base tensorflow/serving
```

## **Copy SavedModel**
```
docker cp /home/model1 serving_base:/models/model1
docker cp /home/model2 serving_base:/models/model2
docker cp /home/model.config serving_base:/models/model.config
```
## **Commit**
```
docker commit serving_base acvision
```
## **Stop Serving Base**
```
docker kill serving base
```
## Checking Docker image
![image](https://user-images.githubusercontent.com/37177883/135968171-deb2c4e3-830a-4dba-8135-2e131e75bf1d.png)

```
docker run --rm --name serve_acvision -p8502:8501 -p8503:8501 -p8504:8501 -d acvision
```

![Screenshot from 2021-10-05 13-46-56](https://user-images.githubusercontent.com/37177883/135968404-689bf09a-95a2-408a-87cd-67c25df7dabd.png)

## **Testing the Server**
![Screenshot from 2021-10-05 13-58-28](https://user-images.githubusercontent.com/37177883/135968720-f2334587-fc78-4d39-9902-beaa7d77f97e.png)

**Comment by singhniraj08** [comment]
@gabbygab1233,

There is no need to mount the models and config files on docker command if you copy your models and configs files to your own serving image.

I tried the below method to create my own serving image and was able to serve my model using own serving image. I also tried with multiple models but it didn't worked for me. [Creating your own image](https://www.tensorflow.org/tfx/serving/docker#creating_your_own_serving_image) works for single model only.

Thank you!


```

docker pull tensorflow/serving
docker run -d --name serving_base tensorflow/serving
docker cp "$(pwd)/models/testdata/saved_model_half_plus_two_cpu" serving_base:/models/saved_model_half_plus_two_cpu
docker commit --change "ENV MODEL_NAME saved_model_half_plus_two_cpu" serving_base tensorflow_serving:2.11.0
docker kill serving_base
docker run -p 8501:8501 -t tensorflow_serving:2.11.0

```

<img width="1496" alt="Screenshot 2023-02-08 at 10 28 40 AM" src="https://user-images.githubusercontent.com/110809167/217439464-88a7b01e-40e9-47af-8250-93595b7d0bc0.png">
